### PR TITLE
Added add-feed script to manually add a feed to the queue

### LINKED ISSUE #273: Write a command line tool for injecting a feed into the queue
In #112, I mentioned a cool idea that I got from [this code](https://github.com/GetStream/Winds/blob/48cda6bef1ae4cd0612368bef86becead4f0e8c5/api/src/commands/winds-og.js) is to add a node.js command-line tool that lets you inject a feed name/URL manually into the running queue.

This would be great for debugging, and for our testing purposes right now, while we wait to get all the feeds loading automatically.

This is somewhat similar to #266, but where that one is an automated process that goes and reads data from the wiki page, this one would be to write a command line tool that works like this:

```
$ node add-feed.js "FirstName LastLastName" https://url.to/user/feed.xml
```

This would cause the user "FirstName LastName", and their feed URL to be inserted into our running feed queue, and make it easier to test.

### PR DESCRIPTION
**Pull request by miggs125**
fix #273. This PR is unfinished but I wanted to create one for feedback before I commit more work into this. I have a working script with some bugs. I've manually tested the script by checking the bull board and confirming the feed job was queued.

To run the script, use ``` npm run add-feed -- --name "bloggers name" --url <feed url> ```. This will make use of the feedQueue.add under ``` src/backend/feed/queue ``` to add a feed to the queue for processing. 

I wanted to make this as an npm script so I changed package.json but if the script is executed using npm, the appropriate error messages seem to be suppressed and it gives a vague npm error message instead. When run as ``` node add-feed.js ..etc ``` the proper error and info messages are displayed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
     "server": "node src/backend/web/server",
     "hint": "hint http://localhost:3000",
     "webhint": "cross-env PORT=3000 npm-run-all -r -p server hint",
-    "debug-server": "nodemon server.js"
+    "debug-server": "nodemon server.js",
+    "add-feed": "node src/backend/utils/add-feed.js"
```
Comment (humphd) [inline package.json]: Let's use `bin` instead, see docs at https://docs.npmjs.com/files/package.json#bin

File: src/backend/utils/add-feed.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const feedQueue = require('../feed/queue');
+
+const log = logger.child({ module: 'add-feed' });
+
+async function add() {
+  if (
+    !Object.prototype.hasOwnProperty.call(args, 'name') ||
+    !Object.prototype.hasOwnProperty.call(args, 'url')
+  ) {
+    log.error('INVALID NUMBER OF ARGUMENTS');
```
Comment (humphd) [inline src/backend/utils/add-feed.js]: Nicer to print a help/usage message when the args are wrong, teach the user how to call it.

File: src/backend/utils/add-feed.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    process.exit(1);
+  }
+  const { name, url } = args;
+  if (!isValidUrl(url)) {
+    log.error('INVALID URL');
+    process.exit(1);
+  }
+  const cleanName = String(name);
+  if (cleanName.match(/[.*+?^${}()|[\]\\`]/)) {
```
Comment (humphd) [inline src/backend/utils/add-feed.js]: What's the goal here?  Why do we need to validate names?  What if someone's name contains unicode or some such?

File: src/backend/utils/add-feed.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    log.error('ONLY ALPHANUMERICAL CHARACTERS ALLOWED');
+    process.exit(1);
+  }
+
+  const feed = { name, url };
+
+  log.info(`Enqueuing feed job for ${url}`);
```
Comment (humphd) [inline src/backend/utils/add-feed.js]: We probably don't need to log when things work, per Unix tradition.

File: src/backend/utils/add-feed.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        delay: process.env.FEED_QUEUE_DELAY_MS || 60 * 1000,
+      },
+      removeOnComplete: true,
+      removeOnFail: true,
+    })
+    .catch(err => {
+      log.error({ err }, 'Error enqueuing feed');
+      process.exit(1);
+    });
+  log.info('feed added');
```
Comment (humphd) [inline src/backend/utils/add-feed.js]: Remove this, I think.

File: src/backend/utils/add-feed.js (hunk 3 above)
Comment (miggs125) [reply src/backend/utils/add-feed.js]: I was over thinking it. Over engineered, will remove

File: src/backend/utils/add-feed.js (hunk 2 above)
Comment (miggs125) [reply src/backend/utils/add-feed.js]: good idea!

**Comment by miggs125** [comment]
@humphd I made changes to reflect the review comments. I haven't added a test yet. Not sure of how much work is involved. I was thinking we could make that a separate issue? I tested it manually and it seems to work. @manekenpix tested and confirmed it works as well. The tool can be used as such 
`add-feed --name "bob dole" --url http://theBestUrlEverDurrr`

File: src/backend/utils/add-feed.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
```
Comment (humphd) [inline src/backend/utils/add-feed.js]: Can this live in `tools/` or something?  It's not really `src/`.

**Pull request merged**